packages/core/phase3/lib/pncUpdateRequestGenerators/normalDisposalGenerator/getNextHearingDateFromOffencesList.ts

```diff
@@ -0,0 +1,11 @@
+import type { Offence } from "../../../../types/AnnotatedHearingOutcome"
+
+const getNextHearingDateFromOffencesList = (offences: Offence[]): Date | undefined => {
```

Comment:
I think we can just call this `getNextHearingDateFromOffences`, no need for `OffencesList`.